<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EgmProtoDemo</class>
 <widget class="QWidget" name="EgmProtoDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EgmProtoDemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>UDP1</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>本地端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUdpPort1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>31414</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUdpUnbind1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>绑定端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUdpBind1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>绑定端口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,2,0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>远程IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRemoteIP1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>远程端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRemotePort1">
          <property name="text">
           <string>31415</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="UdpSendData1">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>0a09080010e990fb6e1803124512430a1b099a9999999999f13f119a99999999990140196666666666660a40122409000000000000f03f110000000000000040190000000000000840210000000000001040</string>
        </property>
        <property name="placeholderText">
         <string>发送数据(HEX):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendUdp1EgmData">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送EgmSensor数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendUdp1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="UdpLog1">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>UDP2</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>本地端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUdpPort2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>31415</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUdpUnbind2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>解绑端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUdpBind2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>绑定端口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2,0,1">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>远程IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRemoteIP2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>远程端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRemotePort2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>31414</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="UdpSendData2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>0a0908011083dcfb6e1800124512430a1b09000000000000f83f110000000000000440190000000000000c40122409000000000000f83f110000000000000440190000000000000c402100000000000012401a4512430a1b09000000000000f83f110000000000000440190000000000000c40122409000000000000f83f110000000000000440190000000000000c40210000000000001240</string>
        </property>
        <property name="placeholderText">
         <string>发送数据(HEX):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendUdp2EgmData">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送EgmRobot数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendUdp2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>发送数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="UdpLog2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
